json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -332,6 +338,17 @@ public function update_settings() {
 					}
 					break;
 
+				case 'holidaysnow':
+					if ( empty( $value ) || WPCOM_JSON_API::is_falsy( $value ) ) {
+						if ( function_exists( 'jetpack_holiday_snow_option_name' ) && delete_option( jetpack_holiday_snow_option_name() ) ) {
+							$updated[ $key ] = false;
+						}
+ 					} else if ( function_exists( 'jetpack_holiday_snow_option_name' ) && update_option( jetpack_holiday_snow_option_name(), 'letitsnow' ) ) {
+						$updated[ $key ] = true;
+ 					}
+
```

Comment:
I believe `break;` is aligned properly, one tab in from the `case`. If you'd prefer, I can get rid of like `349` which is just a newline.